docs/README.md

```diff
@@ -0,0 +1,37 @@
+## The Documentation Website
+
+This folder holds the source code of our documentation website:
+[kotlin.github.io/dataframe](https://kotlin.github.io/dataframe).
+
+It's built using [WriterSide](https://www.jetbrains.com/writerside/) and published
+by a [Github Action](../.github/workflows/main.yml).
+The file structure mirrors the default WriterSide structure with some small exceptions.
+
+Images all README files can be stored in [docs/imgs](./imgs).
```

Comment:
Let's mention the file with page structure, and the fact that all new pages should be inserted into it